pajbot/bot.py

```diff
@@ -716,6 +716,8 @@ def send_message_to_user(
             self.whisper(user, message)
         elif method == "me":
             self.me(message)
+        elif method == "announce":
+            self.announce(message)
```

Comment:
Announcements are announcements, they aren't to a specific user, but to the entire chat. I don't think this is appropriate in this function.